### DISCUSSION TITLE
Question: Is there a way to allow Secure mode with Web Chat?

### DISCUSSION THREAD

**Discussion by SlavaKeshkov** [microsoft/PubSec-Info-Assistant#889] [Q&A]
Dear Microsoft team,

In the documentation you point out that:
Secure Mode is not compatible with the following IA features:
 * Web chat (private endpoints for Bing API services are not available)

Can deployment be considered in a way that Azure resources used for Talk to files and Chat ungrounded utilize private network and Web chat could utilise the public network?

Any downsides to this approach?

Thanks in advance!

**Comment by bjakems** [comment]
Correct, web chat is not supported in 1.2 secure mode.  Please submit an Issue with the enhancement tag in order to record your request.
